### PR TITLE
[MediaCard] Remove extraneous space when card has no actions

### LINKED ISSUE #4534: [MediaCard] Extraneous space when card has no actions
## Issue summary

In vertical view (smaller viewports), MediaCard without `primaryAction` or `secondaryAction` has extra space at the bottom.

## Expected behavior (no extraneous space between description and bottom)

1jxxq csb app_ (4)

## Actual behavior (extraneous space between description and bottom)

1jxxq csb app_ (3)

## Reduced test case

```tsx
      <MediaCard
        title="Turn your side-project into a business"
        description={`In this course, you’ll learn how the Kular family turned their mom’s recipe book into a global business.`}
      >
        <VideoThumbnail
          videoLength={80}
          thumbnailUrl="https://burst.shopifycdn.com/photos/business-woman-smiling-in-office.jpg?width=1850"
        />
      </MediaCard>
```

https://codesandbox.io/s/determined-flower-1jxxq?file=/App.js

## Specifications

- Are you using the React components? (Y/N): Y
- Polaris version number: 7.0.0

### PR DESCRIPTION
**Pull request by emilycritter**
### WHY are these changes introduced?

Fixes #4534

In vertical view (smaller viewports), MediaCard without primaryAction or secondaryAction has extra space at the bottom.

### WHAT is this pull request doing?

This PR adds a conditional to check for actions thus removing an extra, empty div that was resulting in extraneous space at the bottom of the card.

Before:
image showing the card component with extra white space at the bottom

After:
image showing the absence of white space when no action buttons are included

### CODE REVIEW

**Comment by emilycritter** [comment]
Is this project participating in hacktoberfest?

**Comment by kaelig** [comment]
>Is this project participating in hacktoberfest?

Not that I know, but a huge thank you for this contribution!

Could you also add a new example in https://github.com/emilycritter/polaris-react/blob/fix/media-card-extraneous-space/src/components/MediaCard/README.md to illustrate this use-case?

File: src/components/MediaCard/README.md
Comment (kaelig) [inline src/components/MediaCard/README.md]:
```suggestion
Use when media card does not require any actions.
```

File: src/components/MediaCard/README.md
Comment (kaelig) [inline src/components/MediaCard/README.md:221]: This content wouldn't be something we'd want without a call to action, but that's something that can be addressed separately.

**Review by kaelig — APPROVED** [review]

**Comment by kaelig** [comment]
Not sure adding a test is useful here, since visual regression tests would catch any issues with how layout is impacted by future changes.

I'll let the maintainers decide!

File: src/components/MediaCard/MediaCard.tsx
Comment (kyledurand) [inline src/components/MediaCard/MediaCard.tsx:107]: If we had codecov still active it would complain about not adding tests for this and we don't have visual regression testing at the moment so I think it worth adding a couple of tests here. I think we can just test to make sure button group doesn't render if primary or secondary actions are passed in

Comment (kaelig) [reply src/components/MediaCard/MediaCard.tsx:107]: Sounds good! I added a simple test in f729d60

**Review by kyledurand — APPROVED** [review]

**Comment by kaelig** [comment]
Since some of the tests couldn't complete (probably due to the fact the PR comes from a fork), an admin will have to merge this PR.

**Pull request merged**
